### ISSUE TITLE
JN.1.54 subbranch with S:F456L and Orf8:C61*, V62M(83, 4 regions, mainly in Spain)

### ISSUE THREAD

**Issue by Over-There-Is** [sars-cov-2-variants/lineage-proposals#2054]
JN.1.54> C9817T, T22928C(S:F456L), C28076A(Orf8:C61*), G28077A(Orf8:V62M)
Query: C8293T, C8917T, A9085G
6 has no coverage on 28076-28077.
22927-22928 rev is a frequent artefact in Spanish sequences.

Earliest sequence: 03-09 Barcelona, Catalunya, Spain EPI_ISL_19055427
Latest sequence: 07-03 Almeria, Andalusia, Spain EPI_ISL_19418156
Spain(Mainland 73, Canary Islands 2), UK 7, US 1

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_2c28b_ac5230.json?c=country&label=id:node_7283160
![image](https://github.com/user-attachments/assets/8ba1eea4-76e4-40bd-88ec-428623e13551)

<details><summary>Sequences:</summary>

EPI_ISL_19055385, EPI_ISL_19055389, EPI_ISL_19055391,
EPI_ISL_19055427, EPI_ISL_19126142, EPI_ISL_19126554,
EPI_ISL_19141038, EPI_ISL_19141042, EPI_ISL_19143903,
EPI_ISL_19143913, EPI_ISL_19146196, EPI_ISL_19146198,
EPI_ISL_19146200, EPI_ISL_19146209, EPI_ISL_19146217,
EPI_ISL_19146290, EPI_ISL_19146292-19146293, EPI_ISL_19146295-19146298,
EPI_ISL_19146300, EPI_ISL_19146302-19146305, EPI_ISL_19146343,
EPI_ISL_19146356, EPI_ISL_19146358, EPI_ISL_19147058,
EPI_ISL_19151426, EPI_ISL_19151430, EPI_ISL_19163931,
EPI_ISL_19164648, EPI_ISL_19165395, EPI_ISL_19167760,
EPI_ISL_19168106, EPI_ISL_19175169-19175170, EPI_ISL_19175173,
EPI_ISL_19175176, EPI_ISL_19175179, EPI_ISL_19175181,
EPI_ISL_19175184, EPI_ISL_19175190, EPI_ISL_19175517,
EPI_ISL_19176648, EPI_ISL_19185435, EPI_ISL_19193878,
EPI_ISL_19194792, EPI_ISL_19194795, EPI_ISL_19198250,
EPI_ISL_19198253, EPI_ISL_19198256, EPI_ISL_19198341,
EPI_ISL_19201452, EPI_ISL_19201459, EPI_ISL_19210740,
EPI_ISL_19213674, EPI_ISL_19214767, EPI_ISL_19218644,
EPI_ISL_19218978, EPI_ISL_19263373, EPI_ISL_19267694,
EPI_ISL_19267868, EPI_ISL_19267872, EPI_ISL_19267886,
EPI_ISL_19267894, EPI_ISL_19267903-19267904, EPI_ISL_19267906,
EPI_ISL_19277308, EPI_ISL_19322194, EPI_ISL_19322275,
EPI_ISL_19362886, EPI_ISL_19410030, EPI_ISL_19417975-19417976,
EPI_ISL_19417982, EPI_ISL_19418066, EPI_ISL_19418142,
EPI_ISL_19418156
</details>

**Comment by FedeGueli** [comment]
Yeah i was tracking this since early June with C28076A, G28077A,T3565c , it stareted well but died then. I was convinced i had proposed it but it doesnt seem so. anyway it creates Orf8A and Orf8b interesting mutation.
cc @ryhisner i think i sent you this one, earlier this summer.

closing it because it is dead

**Comment by Over-There-Is** [comment]
I still wonder if V62M create a new frame.

**Comment by FedeGueli** [comment]
> I still wonder if V62M create a new frame.

i dont remember what Ryan said about that i think the TRS was quite bad there but i dont recall if he confirmed or not. @ryhisner could take a look please?
